src/test/java/nl/tudelft/pixelperfect/SpaceshipTest.java

```diff
@@ -27,39 +28,53 @@ public void before() {
   }
 
   /**
-   * Test the get health functionality by verifying full health.
+   * Test the getLog functionality by checking the existance of a log.
+   */
+  @Test
+  public void testGetLog() {
+
```

Comment:
This test is empty until further notice, due to the current abscence of log entries.